plugins/outputs/riemann/riemann.go

```diff
@@ -24,8 +26,12 @@ var sampleConfig = `
   url = "localhost:5555"
   ## transport protocol to use either tcp or udp
   transport = "tcp"
-  ## separator to use between input name and field name in Riemann service name
+  ## separator to use between measurement name and field name in Riemann service name
   separator = " "
+  ## set measurement name as a Riemann tag instead of prepending it to the Riemann service name
+  measurement_as_tag = false
+  ## list of Riemann tags, if specified use these instead of any Telegraf tags
+  tags = ["telegraf","custom_tag"]
```

Comment:
@sparrc hmm ok, makes sense. I modified the code to match your proposal. If `riemann_tag_keys` are specified / non-empty it will send only their tag values as Riemann tags, otherwise it will send all tag values.